### ISSUE TITLE
Virtually overclocking hardware for speed gains

### ISSUE THREAD

**Issue by HappyDazes** [jpd002/Play-#306]
This is becoming common among many emulators, especially video game console ones.

I don't know how easy/difficult it would be for Play! to do this, but other high-level emulators have been successful at it, like PPSSPP (PSP emulator). They usually allow the user the option (or pre-defined options) to increase the rate at which the emulated CPU/etc. should work, thus, making games run faster/slower depending on how fast the emulated hardware should be generating code or executing it.

I think that this would be a great short or even long-term feature that will really help games speedup, especially for the Android (ARM) and iOS builds since ARM hardware is significantly slower than Intel.

You can have maybe (double-rate execution; speed everything up 2x) or such or even 150% rate.

This could also work in tandem with https://github.com/jpd002/Play-/issues/71 eventually.

If this is unfeasible for Play! to do, I understand and can close this issue; just trying to suggest ideas.

**Comment by jpd002** [comment]
I don't think it would be too hard to do, except, I'm not sure it'd be very beneficial. AFAIK, this would only help games that idle a lot (not sure FFXII is one of those).

**Comment by Nobbs66** [comment]
It'll help a few games with a variable framerate like Shadow of the Colossus. The performance hit would be massive though.
